features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -467,6 +467,9 @@
         <SAMLDefaultAssertionEncryptionAlgorithmURI>http://www.w3.org/2001/04/xmlenc#aes256-cbc</SAMLDefaultAssertionEncryptionAlgorithmURI>
         <SAMLDefaultKeyEncryptionAlgorithmURI>http://www.w3.org/2001/04/xmlenc#rsa-oaep-mgf1p</SAMLDefaultKeyEncryptionAlgorithmURI>
         <SLOHostNameVerificationEnabled>true</SLOHostNameVerificationEnabled>
+
+        <!-- SAML2 artifact validity period in minutes -->
+        <SAML2ArtifactValidityPeriod>4</SAML2ArtifactValidityPeriod>
```

Comment:
can we change to "SAML2ArtifactValidityPeriodMinutes"

Reply:
How about "SAML2ArtifactValidityPeriod**In**Minutes"?